bitrise.yml

```diff
@@ -10,7 +10,7 @@ trigger_map:
 - push_branch: v8.1.7
   workflow: release
 - pull_request_source_branch: "*"
-  workflow: runParallelTests
+  workflow: L10nBuild
```

Comment:
🚧 temprorary change to test the workflow. To set back to runParallelTests once it is green

Reply:
L10nBuild working correclty with these changes: https://app.bitrise.io/build/70ae3434-034c-4bba-b601-ace4805d2b05#?tab=log